google-ads/pom.xml

```diff
@@ -36,7 +36,7 @@
     <auto-value.version>1.7.3</auto-value.version>
     <!-- Keep all versions below consistent with the io.netty version. -->
     <netty.version>2.0.26.Final</netty.version>
-    <gax.version>1.60.1</gax.version>
+    <gax.version>1.63.0</gax.version>
```

Comment:
Was 1.60.1 used for latency testing? If so, this change will require another round of tests before release.

Reply:
Ah, yeah you're right. Will get a fresh set of numbers on this before releasing.

Reply:
Planning to merge this CL to master and use the master build (already verified that this is up and running).

Reply:
Sounds good. Thanks.